src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,7 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Updated examples in reference documentation for a few `*-AzStorageAccount*` cmdlets to use `-Name` instead of its aliases
```

Comment:
```suggestion
```
Document-only PR is not required to update changelog.
Online documents will be updated in a few days after official release of Azure PowerShell.